pallets/liquidate/src/lib.rs

```diff
@@ -219,11 +220,11 @@ pub mod module {
                                 }
                             };
                         if borrow_currency_amount > 0 {
-                            if currency_price == Price::MIN {
+                            if currency_price == MIN_PRICE {
                                 continue 'outer;
                             }
                             let borrow_currency_sum_price = match borrow_currency_amount
-                                .checked_mul(currency_price)
+                                .checked_mul(currency_price.into_inner())
```

Comment:
FixedU128 should not use `into_inner()` when calculating. You can use `FixedU128::checked_mul_int`.

Reply:
fixed